spec/models/experiment_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Experiment do
+  it { should have_many :scientists }
+end
```

Comment:
Association test missing.

Also, if we're following a TDD approach, the test for `longest_running` should be created before the implementation code.